src/main/java/com/milaboratory/mixcr/cli/ActionAlign.java

```diff
@@ -121,6 +117,20 @@ public void go(ActionHelper helper) throws Exception {
 
         System.out.println("Reference library: " + library.getLibraryId());
 
+        // Printing library level warnings, if specified for the library
+        if (!library.getWarnings().isEmpty()) {
+            System.out.println("Library warnings:");
+            for (String l : library.getWarnings())
+                System.out.println(l);
+        }
+
+        // Printing citation notice, if specified for the library
+        if (!library.getCitations().isEmpty()) {
```

Comment:
Flag to suppress citation required!

Reply:
Connected to #224 and letter we received a week ago about "-q" (quiet) option.

Reply:
Created #307